src/PhpDoc/ResolvedPhpDocBlock.php

```diff
@@ -192,7 +198,9 @@ public function merge(array $parents, array $parentPhpDocBlocks): self
 		// skip $result->nameScope
 		$result->templateTypeMap = $this->templateTypeMap;
 		$result->templateTags = $this->templateTags;
-		// skip $result->phpDocNodeResolver
+		if (isset($this->phpDocNodeResolver)) {
+			$result->phpDocNodeResolver = $this->phpDocNodeResolver;
```

Comment:
Hi, you¨re adding a simple boolean value which there are already many present. These changes to phpDocNode and phpDocNodeResolver are not required to make this work, and cause these build failures.

If the PR doesn't work without these changes, please observe how different boolean values are handled here, it shouldn't be hard to make them work.